test/tests/meshgenerators/break_mesh_by_block_generator/tests

```diff
@@ -149,4 +149,17 @@
                'interface;'
     []
   []
+
+  [hanging_nodes]
+    type = 'Exodiff'
+    input = 'hanging_nodes_parallel.i'
+    exodiff = 'hanging_nodes_parallel_out.e'
+    requirement = 'The system shall properly assign node processor ids for paritioned meshes'
+    design = 'meshgenerators/BreakMeshByBlockGenerator.md'
+    issues = '#15793'
+    mesh_mode = 'REPLICATED'
+    min_parallel = 12
+    max_parallel = 12
+    recover = false
```

Comment:
I believe that you can likely remove this now that we don't have --mesh-only, right?

Reply:
I can't remove 
```    mesh_mode = 'REPLICATED'
    recover = false``` becasue break mesh by block works only in replicated mode and currenlty deos not allow recover
    
also, if I remove 
```
    min_parallel = 12
    max_parallel = 12
```
and run the tests in debug mode nad run in debug with -j12 or -p12 will no trigger the assertion

so all those for lines should stay.

Reply:
This comment was meant only at the recover option (it displays the lines above in addition). Sounds good